storage/validateArtifactLowerCase/validateArtifactLowerCase.groovy

```diff
@@ -0,0 +1,29 @@
+import org.artifactory.build.promotion.PromotionConfig
+import org.artifactory.build.staging.ModuleVersion
+import org.artifactory.build.staging.VcsConfig
+import org.artifactory.exception.CancelException
+import org.artifactory.repo.RepoPathFactory
+import org.artifactory.request.Request
+import org.artifactory.util.StringInputStream
+
+def repositories = new JsonSlurper().parse(new File(ctx.artifactoryHome.haAwareEtcDir, 'plugins/validateArtifactLowerCase.json'))
+storage {
+    beforeCreate {
+        item - >
+            def repositorykey = item.repoPath.repoKey
+        
+        repositories.each {
```

Comment:
Instead of checking every repository in the list, you can just check if the target repoKey is contained in the list. Something like:

```
if (repositoryKey in repositories) {
```

Reply:
@elioengcomp I think its a better way instead looping so i changed it